src/scene/forward-renderer.js

```diff
@@ -813,6 +813,10 @@ Object.assign(ForwardRenderer.prototype, {
         device.setScissor(x, y, w, h);
 
         if (clear) {
+            // use camera clear options if any
+            if (!options)
+                options = camera._clearOptions;
```

Comment:
what's the difference between camera._clearOptions and all the settings used below (camera._clearColor et al)?

Reply:
There are two ways .. one is settings on camera, one is override settings on camera. 
Don't ask ... a lot of this code seems over engineered to me.

Reply:
Ok I take the question back!